client/src/components/organisms/ProductContent.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import Carousel from 'react-material-ui-carousel';
+import styled from '@emotion/styled';
+import ImageBox, { EImageSize } from '@components/molecules/ImageBox';
+import { ProductStatus, TProductDetail } from '@fleamarket/common';
+import Dropdown from '@components/molecules/Dropdown';
+import {
+  COLOR,
+  SCROLLBAR_THUMB,
+  TEXT_LARGE,
+  TEXT_LINK_SMALL,
+  TEXT_MEDIUM,
+  TEXT_X_SMALL,
+} from '@constants/style';
+import { getTimeGapString } from '@utils/time';
+
+interface IProps {
+  details: TProductDetail;
+}
+
+const ProductContent: React.FC<IProps> = ({ details }) => {
+  const {
+    title,
+    content,
+    images,
+    status,
+    categoryName,
+    createdAt,
+    rooms,
+    likes,
+    hit,
+    userName,
+    locationName,
+  } = details;
+
+  return (
+    <>
+      <Carousel
+        className='carousel'
+        indicatorContainerProps={{
+          style: {
+            position: 'relative',
+            marginTop: '-40px',
+            paddingBottom: '18px',
+            zIndex: 10,
+          },
+        }}
+      >
+        {images?.map((url, index: number) => (
+          <ImageBox
+            key={index}
+            src={url}
+            alt={'products'}
+            type={EImageSize.full}
+          />
+        ))}
+      </Carousel>
+
+      <ContentDiv>
+        <Dropdown
+          dropDownList={Object.values(ProductStatus).filter(
+            (k) => k !== status,
+          )}
+          handleClick={(value) => {
+            return value;
+          }}
+        >
+          <DropdownDiv>{status}</DropdownDiv>
+        </Dropdown>
+        <h3 className='title'>{title}</h3>
+        <span className='sub_title'>
+          {categoryName}
+          <span>∙</span>
+          {getTimeGapString(new Date(createdAt ?? 0))}
+        </span>
+        <p className='content'>{content}</p>
+        <span className='sub_title'>
+          채팅 {rooms}
+          <span>∙</span>
+          관심 {likes}
+          <span>∙</span>
+          조회 {hit}
+        </span>
+        <WriterInfoDiv>
+          <h4 className='guide'>판매자 정보</h4>
+          <div className='user'>{userName}</div>
+          <span className='location'>{locationName}</span>
```

Comment:
```suggestion
        <WriterInfoDiv>
          <h4 className='guide'>판매자 정보</h4>
          <div className='user'>{userName}</div>
          <span className='location'>{locationName}</span>
```

재사용을 하지 않는 부분이라 상관은 없지만 컴포넌트로 분리해도 될 것 같다는 생각을 했습니다!